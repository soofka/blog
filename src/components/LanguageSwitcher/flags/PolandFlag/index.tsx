import * as React from 'react';
import { FlagPropsInterface } from '../../index';

export const PolandFlag = (props: FlagPropsInterface) => (
  <svg width={100} height={60} viewBox="0 0 16 10" {...props}>
    <path fill="#fff" d="M0 0h16v10H0z" />
    <path fill="#dc143c" d="M0 5h16v5H0z" />
  </svg>
);

export default PolandFlag;
