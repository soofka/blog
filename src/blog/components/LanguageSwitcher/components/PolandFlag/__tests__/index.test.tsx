import * as React from 'react';
import { shallow } from 'enzyme';

import PolandFlag from 'components/LanguageSwitcher/components/PolandFlag';

describe('<PolandFlag/>', () => {

  it('renders correctly', () => {
    expect(shallow(<PolandFlag/>)).toMatchSnapshot();
  });

  it('renders correctly with optional props', () => {
    expect(shallow(
      <PolandFlag
        className="testClassName"
      />,
    )).toMatchSnapshot();
  });

});
