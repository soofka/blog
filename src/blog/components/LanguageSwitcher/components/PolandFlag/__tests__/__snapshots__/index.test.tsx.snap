// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PolandFlag/> renders correctly 1`] = `
<svg
  height={60}
  viewBox="0 0 16 10"
  width={100}
>
  <path
    d="M0 0h16v10H0z"
    fill="#fff"
  />
  <path
    d="M0 5h16v5H0z"
    fill="#dc143c"
  />
</svg>
`;

exports[`<PolandFlag/> renders correctly with optional props 1`] = `
<svg
  className="testClassName"
  height={60}
  viewBox="0 0 16 10"
  width={100}
>
  <path
    d="M0 0h16v10H0z"
    fill="#fff"
  />
  <path
    d="M0 5h16v5H0z"
    fill="#dc143c"
  />
</svg>
`;
